common/views/components/Divider/Divider.tsx

```diff
@@ -7,7 +7,7 @@ type Props = {
   isStub?: boolean;
 };
 
-const Rule = styled.hr<Props>`
+const Rule = styled.div<Props>`
```

Comment:
The case for `hr` isn't a strong one. If we _really_ find we're missing something semantic, we could make this component optionally output an `hr` (e.g. with an `isHr` prop). But I'm not sure it's required and there are almost certainly better ways of handling section semantics if we need them.